build/rbac/rbac.yaml

```diff
@@ -80,7 +80,7 @@ rules:
     resources:
```

Comment:
Mostly this is a lot of things just changing the place where they live. 

Notably, I changed the PSP name, but since all resources are updated at the same time, it shouldn't cause any issues. 

I also added `# namespace:{cluster,operator}` comments to all helm resources. They were helpful in making sure that resources went to the right places and will be good for generating `common.yaml` later.

There are some minor differences in ordering of parameters in some of the manifests.

The overview of what actually changed can be gleaned from this generated file.